atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -438,11 +438,17 @@ def __init__(self, signing_intents, default_signing_intent):
         self.get_signing_intent_by_name(self.default_signing_intent)
 
     def get_signing_intent_by_name(self, name):
+        valid = []
         for entry in self.signing_intents:
-            if entry['name'] == name:
+            this_name = entry['name']
+            if this_name == name:
                 return entry
 
-        raise ValueError('unknown signing intent name "{}"'.format(name))
+            valid.append(this_name)
+
+        raise ValueError('unknown signing intent name "{}", valid names: {}'
+                         .format(name,
+                                 ', '.join(valid)))
```

Comment:
Nit: can we get the format on a single line?

Reply:
Updated.